# Excepciones

"""Try = contiene el codigo que se va a intentar ejecutar y que podria lanzar una excepcion,
Si ocurre una excepcion dentro del bloque try el flujo de ejecucion se detiene y se transfiere 
al bloque catch"""

"""Catch = Atrapa la excepcion que se dio en try y la maneja, podes registrar el error, mandar 
mensajes, etc"""


"""Finally = Contiene el codigo que se ejecuta siempre, independiemente de si se lanzo
o no una excepcion, esto permite hacer limpieza, como cerrar conexiones de archivor,etc,etc,etc"""




    